changelog/pending/20230622--cli-new--pulumi-new-s-org-project-stack-checks-the-specified-organization-for-project-existence-rather-than-the-currentuser.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: cli/new
+  description: `pulumi new -s 'org/project/stack'` checks the specified organization for project existence rather than the currentUser.
```

Comment:
What about the default org? Will it use the default org, rather than the current user, if one is set?

https://www.pulumi.com/docs/cli/commands/pulumi_org_set-default/

Reply:
It won't we'd need a call to `workspace.GetBackendConfigDefaultOrg` to work out the default org.

Reply:
To be clear, it should be to the effect:
```typescript
const org = explicitOrg || defaultOrg || currentUser;
```

Correct?